Assets/Scripts/ItemType.cs

```diff
@@ -0,0 +1,7 @@
+public enum ItemType
+{
+    other,

```

Comment:
```suggestion
    none,
```
Andere verwoording die logischer kan zijn